components/AppBar/src/MDCAppBarNavigationController.m

```diff
@@ -29,6 +29,9 @@ @interface MDCAppBarNavigationControllerInfo : NSObject
 
 @end
 
+@interface MDCAppBarNavigationController () <UIGestureRecognizerDelegate>
```

Comment:
Please document this category. I assume this is necessary to assign itself as the `interactivePopGestureRecognizer`?

Reply:
We do not require documentation of internal categories.

Reply:
The Google Style Guide says [anything "non-trivial" should be documented](http://google.github.io/styleguide/objcguide.html#declaration-comments).   Do you mean that you feel this is a trivial interface declaration?  I personally questioned why it wasn't in the public declaration of the interface as I was reading.  I still don't know why it shouldn't just be declared public.  I think documenting why it shouldn't be exposed is one good use of the documentation for this category.


> Every non-trivial interface, public and private, should have an accompanying comment describing its purpose *and how it fits into the larger picture*. (emphasis added)

Reply:
I do feel it is reasonably common practice to keep internal protocol conformances private to the class unless otherwise needed to be exposed publicly. Any comment I would write here would be to the effect of "This is intentionally a private protocol conformance in order to reduce public commitments to specific API contracts". Do you feel that that is necessary here?

Reply:
I would appreciate if you added a comment like that, yes. For me it's explaining that although it requires extra complexity and mental energy to understand the class's conformance, it's worth it to balance keeping this detail private.  If there's a specific risk of clients depending on this protocol, it's worth calling-out as well.

Feel free to make it a non-doc comment, since I don't believe the Style Guide requires and in this case it's largely an explanation to maintainers rather than clients.

Reply:
sgtm; thanks Rob!